src/fragments/lib/flutter.mdx

```diff
@@ -1,10 +1,12 @@
+## Amplify Flutter
+
 <Callout warning>
 
 Welcome to the Amplify Flutter documentation. To stay up to date with the latest changes and provide feedback, please take a look at our [Github repo](https://github.com/aws-amplify/amplify-flutter) or join us on [Discord](https://discord.gg/jWVbPfC).
 
-</Callout>
+Amplify Flutter currently supports iOS/Android as target platforms. Flutter [Web](https://github.com/aws-amplify/amplify-flutter/issues/234) and [Desktop](https://github.com/aws-amplify/amplify-flutter/issues/133) support are future roadmap initiatives that are being tracked in our Github repo.
```

Comment:
Can we change the callout **future roadmap initiatives** to _feature requests_ ?

Reply:
addressed this now!